manifests/pnp4nagios/service.pp

```diff
@@ -0,0 +1,14 @@
+class icinga2::pnp4nagios::service inherits icinga2::pnp4nagios::params {
+
+  if $service_manage == true {
+    service {$service_name:
```

Comment:
here you are assuming the $service_name comes from the params.pp. But what happens if the use change the service name in the pnp4nagios class? will the change in service name gets reflected here?